README.md

```diff
@@ -13,25 +13,24 @@ Player Service is a backend application that serves baseball player data. In add
 ## 🛠️ Setup Instructions
 
 1. Verify system dependencies
-   1. Java 17
+   1. Java >= 17
```

Comment:
I am not sure about this and/or maven version below - do they have to be 17 exactly?